webpack.config.renderer.dev.js

```diff
@@ -50,6 +50,23 @@ export default merge.smart(baseConfig, {
   module: {
     rules: [
       {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+            plugins: [
```

Comment:
Maybe `class-properties` and `es2015-classes` plugins can also included here then we can add comments for why they are used?